scipy/ndimage/interpolation.py

```diff
@@ -72,15 +72,15 @@ def spline_filter1d(input, order=3, axis=-1, output=numpy.float64,
 
     Notes
     -----
-    All functions in `ndimage.interpolation` internally use a b-spline
-    representation of the input image. Whenever using b-splines of
-    `order > 1`, the input image values have to be converted to b-spline
-    coefficients first, which is done by applying this one-dimensional
-    spline filter sequentially along all axes of the input. All of the
-    functions that require such an input will automatically filter their
-    inputs, a behavior controllable through the `prefilter` keyword
-    argument. For functions that accept a `mode` parameter, the results
-    will only be correct if it matches the `mode` used when filtering.
+    All functions in `ndimage.interpolation` do spline interpolation of
+    the input image. If using b-splines of `order > 1`, the input image
```

Comment:
Maybe, "interpolate the input image using b-splines. When `order > 1`,  the input image values are replaced by b-spline coefficients by applying ... "